fmn/sender/consumer.py

```diff
@@ -14,20 +13,10 @@
 
 class Consumer:
     def __init__(self, config, handler):
-        self._url = URL(config["amqp_url"])
+        self._url = get_url_from_config(config)
         self._destination = config["queue"]
         self._handler = handler
         self._url = self._url.update_query(connection_name=f"FMN sender on {self._destination}")
-        if "tls" in config:
-            self._url = self._url.update_query(auth="EXTERNAL")
-            self._url = self._url.update_query(
-                SSLOptions(
-                    cafile=config["tls"]["ca_cert"],
-                    certfile=config["tls"]["certfile"],
-                    keyfile=config["tls"]["keyfile"],
-                    no_verify_ssl=ssl.CERT_REQUIRED,
-                )
-            )
```

Comment:
Is this all obsolete now, does TLS just work without it?

Reply:
I moved that to `fmn.core.amqp.get_url_from_config()` because the consumer's consumer (yeah...) use it too.